charts/authentik/templates/server-deployment.yaml

```
@@ -1,44 +1,37 @@
-{{- range list "server" "worker" }}
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ printf "%s-%s" (include "common.names.fullname" $) . }}
+  name: {{ printf "%s-%s" (include "authentik.names.fullname" $) "server" }}
   labels:
-    {{- include "common.labels" $ | nindent 4 }}
-    app.kubernetes.io/component: "{{ . }}"
+    {{- include "authentik.labels" $ | nindent 4 }}
+    app.kubernetes.io/component: "server"
   {{- if $.Values.annotations }}
   annotations:
     {{- toYaml $.Values.annotations | nindent 4 }}
   {{- end }}
 spec:
-  {{ if eq . "server" -}}
   replicas: {{ $.Values.replicas }}
```

```suggestion
  replicas: {{ .Values.replicas }}
```
We could replace those as the `range` is not there anymore. I don't think it makes a difference though

---

I meant everywhere in the file, but it's not necessary at all